src/styles/main.scss

```diff
@@ -0,0 +1,82 @@
+$border-size: 1px;
+$day-size: 100px;
+$gap-size: 1px;
+$padding-size: 0;
+$font-size: 30px;
+$font-family: Arial, sans-serif;
+$calendar-columns: 7;
+$hover-color: #ffbfcb;
+$hover-move: -20px;
+$transition-duration: 0.5s;
+$start-days: (sun, mon, tue, wed, thu, fri, sat);
+$start-day-offsets: (
+  mon: 0,
+  tue: calc(#{$day-size} + #{$gap-size}),
+  wed: calc(2 * (#{$day-size} + #{$gap-size})),
+  thu: calc(3 * (#{$day-size} + #{$gap-size})),
+  fri: calc(4 * (#{$day-size} + #{$gap-size})),
+  sat: calc(5 * (#{$day-size} + #{$gap-size})),
+  sun: calc(6 * (#{$day-size} + #{$gap-size})),
+);
+
+body {
+  min-height: 100vh;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  flex-flow: wrap;
+  gap: 1px;
```

Comment:
Consider using the `$gap-size` variable instead of hardcoding the value `1px` for the `gap` property. This will make your code more maintainable and consistent with the rest of your SCSS file.